app/add_some_abilities/add_potion_abilities.py

```diff
@@ -0,0 +1,3 @@
+def add_ability_from_potion(knight: dict, ability: str) -> None:
+    if ability in knight["potion"]["effect"]:
+        knight[ability] += knight["potion"]["effect"][ability]
```

Comment:
There is a potential KeyError if `ability` is not already a key in the `knight` dictionary. Consider using `knight.get(ability, 0)` to safely handle cases where the ability might not exist yet.